waveorder/stokes.py

```diff
@@ -0,0 +1,338 @@
+import numpy as np
+
+"""
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
+
+1) A forward function group: 
+
+A = A_matrix(swing, scheme="5-State")
+s0, s1, s3, s3 = s0123_CPL_after_ADR(ret, ori, tra, dop)
+s0, s1, s2 = s012_CPL_after_AR(ret, ori, dop)
+
+2) An inverse function group:
+
+inverse_s0123_CPL_after_ADR(s0, s1, s2, s3)
+inverse s012_CPL_after_AR(s0, s1, s2)
+AR_mueller_from_CPL_projection(s0, s1, s2, s3)
+
+3) A convenience function group:
+
+M = inv_AR_mueller_from_CPL_projection(s0, s1, s2, s3)
+y = matmul(A, x)
+
+----------
+
+Usage: 
```

Comment:
Minor styling issue:
Module docstring can also use NumPy-style and get auto-rendered in docs pages.
E.g. this section can be:
```docs
Examples
--------
Some text
    >>> print(1 + 3)  # some doctest code
    4
```

Reply:
And they should also be rendered in supporting editors when imported. For example in VS code I can see the docstring of the NumPy module imported in this file.

Reply:
Cool! Thanks for showing me this. Fixed.